### ISSUE TITLE
Relation for issues

### ISSUE THREAD

**Issue by dbruggner** [gitbucket/gitbucket#1619]
## Issue
**Impacted version**: not relevant 

**Deployment mode**: not relevant

**Problem description**:
- Addition for issue wishlist: https://github.com/gitbucket/gitbucket/wiki/Wishlist
- As a user of Gitbucket I want to add relations between issues:
  - is duplicate of/ is duplicated by
  - is child of/ is parent of
  - related to
  - blocks/ is blocked by
  - precedes/ follows
